server/src/routes/api/v1/BookListRouter.js

```diff
@@ -21,16 +22,37 @@ BookListRouter.get("/", async (req, res) => {
   }
 });
 
-BookListRouter.get("/:id", async (req, res) => {
+bookListRouter.get("/:id", async (req, res) => {
   try {
     const { id } = req.params;
     const bookList = await BookList.query().findById(id);
-    const serializedBookList = await BookListSerializer.getSummary(bookList);
-
-    return res.status(200).json({ bookList: serializedBookList });
+    debugger;
+    bookList.books = await bookList.$relatedQuery("books");
+    debugger;
+    return res.status(200).json({ bookList });
   } catch (err) {
+    debugger;
     return res.status(500).json({ err });
```

Comment:
remove extra debuggers. this feature should also work as implemented, and you can merge without a serializer, but it would be nice to add back in in the future!

Reply:
in fact, it looks like you have one already, so we might as well use it here!